### PR TITLE
Add decode_json_fields processor to Beats config

### PR DESCRIPTION
**Pull request by jdragonbae**
This commit means adding below info to config.full.yml
`processors:
 - decode_json_fields:
     fields: ["field1", "field2", ...]
     process_array: false
     max_depth: 1
     target: ""
     overwrite_keys: false`

issues #3513

### CODE REVIEW

**Comment by jdragonbae** [comment]
Please, take a look.

File: libbeat/_meta/config.reference.yml [hunk 1]
```diff
@@ -116,8 +116,8 @@
 #
 #   event -> filter1 -> event1 -> filter2 ->event2 ...
 #
-# The supported processors are drop_fields, drop_event, include_fields, and
-# add_cloud_metadata.
+# The supported processors are drop_fields, drop_event, include_fields, 
+# decode_json_fields and add_cloud_metadata.
```
Comment (andrewkroh) [inline libbeat/_meta/config.reference.yml]: Could you please add a comma before the `and`. We usual go for the Oxford comma in our docs and config.

**Comment by jdragonbae** [comment]
@ph 
Thank you for your review. 
I encounter some problem when run `make update` at root directory.
Could you give tips about below error?

```
go install github.com/elastic/beats/vendor/github.com/magefile/mage
make[1]: [mage] Error 127 (ignored)
bash: mage: command not found
make[1]: *** [fields] Error 127
make: *** [update] Error 1
```

File: libbeat/_meta/config.reference.yml (hunk 1 above)
Comment (jdragonbae) [reply libbeat/_meta/config.reference.yml]: Thank you! I'll take it.

**Comment by andrewkroh** [comment]
> Could you give tips about below error?

This is usually caused by not having added your $GOPATH/bin dir to your PATH. I have this in my .bashrc.

```
export GOPATH=$HOME/go
export PATH=$HOME/bin:$GOPATH/bin:$PATH
```

**Pull request merged**